packages/esm-patient-list-management-app/src/lists-dashboard/lists-dashboard.component.tsx

```diff
@@ -73,7 +73,10 @@ const ListsDashboard: React.FC = () => {
               <Tab className={styles.tab}>{t('starredLists', 'Starred lists')}</Tab>
               <Tab className={styles.tab}>{t('systemLists', 'System lists')}</Tab>
               <Tab className={styles.tab}>{t('myLists', 'My lists')}</Tab>
-              <Tab className={styles.tab}>{t('allLists', 'All lists')}</Tab>
+              {/* data-testid is essential for joyride in onboarding app ! */}
+              <Tab data-testid="allListsTab" className={styles.tab}>
+                {t('allLists', 'All lists')}
+              </Tab>
```

Comment:
I'm more in favour of using a semantic HTML `id` attribute here instead of `data-testid` for targeting this element:

```suggestion
              <Tab id="patient-lists-all-tab" className={styles.tab}>
                {t('allLists', 'All lists')}
              </Tab>
```

And then in the `tutorialData` config, we could access it using:

```ts
            target: '#patient-lists-all-tab',
```

The rationale for that is:

- `data-testid` is intended for automated testing, not user interactions. 
- `id` is the canonical HTML attribute for unique element identification
- Separates testing infrastructure from tutorial functionality
- More semantic and universally supported

Reply:
I will make the change asap , Thanks !